frontend/src/metabase/parameters/components/widgets/DateMonthYearWidget.jsx

```diff
@@ -1,21 +1,18 @@
-import React, { Component } from "react";
-
+import React from "react";
 import YearPicker from "./YearPicker";
+import { Flex } from "grid-styled";
 
 import moment from "moment";
 import _ from "underscore";
 import cx from "classnames";
 
-export default class DateMonthYearWidget extends Component {
+export default class DateMonthYearWidget extends React.Component {
   constructor(props, context) {
     super(props, context);
 
     const initial = moment(this.props.value, "YYYY-MM");
     if (initial.isValid()) {
-      this.state = {
-        month: initial.month(),
-        year: initial.year(),
-      };
+      this.state = { month: initial.month(), year: initial.year() };
```

Comment:
I prefer listing each state property on it's own line but that's pretty minor.

Reply:
Yeah me too. I think this might've been a `prettier` artifact but I'll see if I can coerce it back.